repositories/repositories.bzl

```diff
@@ -120,6 +120,14 @@ def repositories():
             urls = ["https://download.docker.com/linux/ubuntu/gpg"],
         )
 
+    # Docker gpg key necessary to install Docker in the containers.
```

Comment:
It looks like it's the same key for all three. Maybe we can just use one.

Reply:
done